src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.html

```diff
@@ -46,20 +46,50 @@
       (searchString)="filterSearch($event)"></sg-search-bar>
   </div>
   <div class="options-section">
-    <button
-      *ngFor="let item of displayedItems"
-      mat-menu-item
-      role="menuitemcheckbox"
-      (click)="toggleSelection($event, item)">
-      <div class="menu-item-content">
-        <span class="menu-item-text">{{
-          displayField ? item[displayField] : item
-        }}</span>
-        <mat-checkbox
-          [color]="'primary'"
-          [checked]="isInSelection(item)"></mat-checkbox>
+    <ng-container
+      *ngIf="
+        displayCategories && displayedItems && displayedItems.length > 0;
+        else noCategory
+      ">
+      <div
+        *ngFor="let categoryGroup of displayedItems"
+        (click)="$event.stopPropagation()">
```

Comment:
Not sure I understand why we are adding this here, a `div` does not have a default click action.
Should this be on the `button` instead?

Reply:
Updated, I added it to the expander. Thank you Pablo!